### PR TITLE
SDK Updates

### PR DESCRIPTION
**Pull request by mediumTaj**
### Summary
This pull request updates the Unity SDK with the latest changes. `Actions` were added to `MessageResponse` and `OutputData` and Assistant integration tests were updated.

For ICP when authenticating with an ICP apikey we authenticate using basicauth.

New language models were added to Speech to Text but since we do not have Language Enums in Unity (Unity enums can only be int) there are no changes in the SDK.

### CODE REVIEW

File: Scripts/Utilities/Credentials.cs [hunk 1]
```diff
@@ -146,7 +147,7 @@ public Credentials(TokenOptions iamTokenOptions, string serviceUrl = null)
         #region SetCredentials
         private void SetCredentials(string username, string password, string url = null)
         {
-            if (username == APIKEY_AS_USERNAME)
+            if (username == APIKEY_AS_USERNAME && !password.StartsWith(ICP_PREFIX))
```
Comment (mamoonraja) [inline Scripts/Utilities/Credentials.cs]: can we add a test when using `ICP_Prefix`?

Comment (mediumTaj) [reply Scripts/Utilities/Credentials.cs]: Unfortunately Unity doesn't have a good testing framework. Right now I am only able to do integration tests with the services. I may be able to start testing now that modern .NET is available for Unity but I'll have to find some time to investigate this.

**Comment by mediumTaj** [comment]
Holding off on release until https://github.com/watson-developer-cloud/unity-sdk/issues/447 is resolved.

**Comment by mediumTaj** [comment]
Issue resolved - merging!

**Pull request merged**